src/app/Bounty/page.tsx

```diff
@@ -0,0 +1,150 @@
+'use client';
+import { useState, useEffect } from 'react';
+import { Button } from '@/components/ui/button';
+import PaymentMethodsDropdown from '@/components/PaymentDropdown';
+import { SolanaAddress, UpiId, BountySubmission } from '@prisma/client';
+import BountySubmissionDialog from '@/components/bounty/BountySubmissionDialog';
+import { getPayoutMethods } from '@/actions/payoutMethods';
+import { getUserBounties } from '@/actions/bounty';
+import { toast } from 'sonner';
+import { Skeleton } from '@/components/ui/skeleton';
+
+export default function Page() {
+  const [isBountyDialogOpen, setIsBountyDialogOpen] = useState<boolean>(false);
+  const [upiAddresses, setUpiAddresses] = useState<UpiId[] | undefined>([]);
+  const [solanaAddresses, setSolanaAddresses] = useState<
+    SolanaAddress[] | undefined
```

Comment:
same as above

Reply:
Thought process what that if user hasn't entered the address then it stays undefined, but youre right we check if address exists anyway so theres no point in this. ill remove them both.